crates/biome_service/src/diagnostics.rs

```diff
@@ -269,6 +296,16 @@ Use the `files.maxSize` configuration to change the maximum size of files proces
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Diagnostic)]
+#[diagnostic(
+    category = "project",
+    message(
+        message("Biome attempted to perform an operation on the registered project, but no project was registered"),
+        description = "This is a bug in Biome. If this problem persists, please report here: https://github.com/biomejs/biome/issues/"
```

Comment:
Message and description are the same but for different purposes. `description` is meant for all those mediums that don't support `biome_console::fmt`, for example the LSP client. This means that `message` and `description` always need to contain the same wording. 

Use `MessageAndDescription` if you don't want to maintain the two of them.